tests/test_closing.py

```diff
@@ -465,7 +465,7 @@ def test_closing_fee(node_factory, bitcoind, chainparams):
         'funder_feerate_per_kw': 29006,
         'fundee_feerate_per_kw': 27625,
         'close_initiated_by': 'funder',
-        'expected_close_fee': 20333
+        'expected_close_fee': 33533 if chainparams['elements'] else 20333
```

Comment:
How is this calculated ? Asking for ~~a friend~~ #3592 where I wondered how much to bump it in functional tests (e.g. https://github.com/ElementsProject/lightning/pull/3592/commits/616a432bd03359afc3abdecb5bbe33b8ced3ccda#diff-d03575dee065f2dc319cf0e8d39ebe95R548)

Reply:
I usually just binary search until I find params that work on both sides (keeping as many parameters as possible fixed).

I have not found a good way to derive from first principles, other than reimplementing the logic in python, which'd just double the maintenance effort without a major upside.